shell/platform/android/build.gradle

```diff
@@ -0,0 +1,42 @@
+// This file only exists to provide IDE support to Android Studio.
+//
+// See ./README.md for details.
+
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:7.0.4"
+    }
+}
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+apply plugin: "com.android.library"
+
+android {
+    compileSdk 31
```

Comment:
This should really be 34. If you can make that change and it does not break please do so. If it breaks just let me know here and merge as is.

Reply:
Should work fine! Done.